test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
Still don't have the big picture on actioncable and this discussion is out of scope of this PR. I will play more with actioncable in the coming days and come back with an issue or a PR if needed. Basically now it seems that whenever the client sends an action to the channel we assume that everything was ok on the server-side. Maybe it's a websockets conceptual thing or maybe developers should handle this logic. We will see :)